src/oslo_aero_depos.F90

```diff
@@ -717,6 +718,12 @@ subroutine oslo_aero_depos_wet ( lchnk, ncol, psetcols, pmid, pdel, q, t, &
                 jnv = 0  !==> below cloud scavenging coefficients are zero (see above)
              endif
 
+! OS 240207 Increase scavenging efficiency for large soluble particles.
+              if (lphase==1.and.mm.eq.l_ss_a2.or.mm.eq.l_ss_a3.or.mm.eq.l_so4_a2) then
```

Comment:
This seems like odd logic. Is it okay to make these changes even if `lphase` is _not_ `1`? That what is what you have. If `lphase` must be `1`, see this alternative.
```suggestion
              if ((lphase==1).and.((mm==l_ss_a2).or.(mm==l_ss_a3).or.(mm==l_so4_a2))) then
```